apps/transport/priv/repo/migrations/20240422145836_create_feedback.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Transport.Repo.Migrations.CreateFeedback do
+  use Ecto.Migration
+
+  def change do
+    create table(:feedback) do
+      add(:rating, :string, null: false)
+      add(:explanation, :string, null: false)
```

Comment:
La taille max sera de 255 caractères en BDD ainsi. C'est cohérent avec le formulaire ? Sinon augmenter la taille, mettre un attribut de taille dans le textarea en HTML et dans le changeset ?

Reply:
Alors, j’ai fait ceci, je veux bien ton avis, j’hésitais un peu entre les types PostgreSQL : https://github.com/etalab/transport-site/pull/3894/commits/92c4a9366e4df8a783d9efbb7733310339726361

J’ai lu ça https://stackoverflow.com/questions/4848964/difference-between-text-and-varchar-character-varying il semblait y avoir un consensus pour dire que text est une bonne idée. Du coup, avec une contrainte dans le Schéma Ecto et pas en BDD sur la longueur. Attention : si on dépasse la longueur autorisée, on se prend une erreur générique, c’est pour ça que j’ai mis la limite assez loin (à 10 000 caractères), je pense que c’est raisonnable de pas coder un avertissement spécifique.